data/net/minecraft/commands/arguments/GameModeArgument.mapping

```diff
@@ -2,6 +2,8 @@ CLASS net/minecraft/commands/arguments/GameModeArgument
 	METHOD getGameMode (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/world/level/GameType;
 		ARG 0 context
 		ARG 1 name
+	METHOD lambda$static$0 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 unknownGameType
```

Comment:
```suggestion
		ARG 0 invalidGameType
```

Based on the field `ERROR_INVALID` and translation `"argument.gamemode.invalid"` using `invalid`, not `unknown`.